frontend/src/app/(routes)/transfers/components/TransfersPage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from '@/custom-hooks/StateHooks';
+import { RootState } from '@/store/store';
+import { getBalances } from '@/store/features/bank/bankSlice';
+import MainTopNav from '@/components/MainTopNav';
+import TransfersHistory from './TransfersHistory';
+import { TRANSFERS_TAB2 } from '../../../../utils/constants';
+import { SINGLE_TAB_TEXT, TRANSFERS_TAB1 } from '@/utils/constants';
+import SingleTransfer from './SingleTransfer';
+import MultiTransfer from './MultiTransfer';
+
+export interface TransfersTab {
+  current: string;
+  to: string;
+}
+const TransfersPage = ({ chainIDs }: { chainIDs: string[] }) => {
+  const [tab, setTab] = useState<TransfersTab>(TRANSFERS_TAB1);
+  const changeTab = (tab: TransfersTab) => {
+    if (tab === TRANSFERS_TAB1) setTab(TRANSFERS_TAB2);
+    else setTab(TRANSFERS_TAB1);
+  };
+  const dispatch = useAppDispatch();
+  const networks = useAppSelector((state: RootState) => state.wallet.networks);
+
+  useEffect(() => {
+    chainIDs.forEach((chainID) => {
```

Comment:
I noticed that a lot of modules use the same code. I also used it in the gov module. I suggest creating a utility function that can retrieve the basic required chain information based on the passed chain ID. This function could be reused across all modules, eliminating the need to write the same code multiple times.